build/vue-loader.conf.js

```diff
@@ -4,17 +4,40 @@ const utils = require('./utils')
 const config = require('../config')
 const isProduction = process.env.NODE_ENV === 'production'
 
-module.exports = {
-  loaders: utils.cssLoaders({
-    sourceMap: isProduction
-      ? config.build.productionSourceMap
-      : config.dev.cssSourceMap,
-    extract: isProduction
-  }),
-  transformToRequire: {
-    video: 'src',
-    source: 'src',
-    img: 'src',
-    image: 'xlink:href'
+module.exports = function vueLoaderConfig(env) {
+  let tsLoader = null;
+  switch (env) {
+    case 'development':
+      tsLoader = 'babel-loader!ts-loader!tslint-loader?{emitErrors:false,typeCheck:false}';
+      break;
+
+    case 'production':
+      tsLoader = 'babel-loader!ts-loader!tslint-loader?{emitErrors:true,typeCheck:false}';
+      break;
+
+    case 'test':
+      tsLoader = 'babel-loader!ts-loader';
+      break;
+
+    default:
+      throw new Error(`No vue loader config for ${env} environment`);
```

Comment:
TODO:
- Add template to README 
- Figure out env files in dev environments and CI 

(don't worry about this for now, we can create issues for this)

Reply:
After going to go fix this I realized this has nothing to do with the `.env` file, which actually isn't a thing in this repo

Reply:
This is a webpack specific environment variable :)